libs/core/src/components/sage-chip/stories/sage-chip-docs.stories.mdx

```diff
@@ -0,0 +1,101 @@
+import { Meta, Story, Canvas, Source, ArgsTable } from '@storybook/addon-docs';
+import { html, render } from 'lit-html';
+
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+
+import * as stories from './sage-chip.stories.js';
+
+<Meta title="components/Chip" component="sage-chip" argTypes={extractArgTypes('sage-chip')} />
+
+# Chip
+
+Chips are used to inform users of the status of an object or to relate properties to a parent. Commonly used in tabular data to indicate status.
+
+## Properties
+
+<ArgsTable of="sage-chip" />
+
+### Default
+
+<Story story={stories.Default} />
+
+<Canvas>
+  <sage-chip label="Label" />
+</Canvas>
+
+### Sentiment
+
+The `sentiment` property represents the named color scheme of the chip component.
+
+<Story story={stories.Sentiment} />
+
+<Canvas>
+  <sage-chip label="Accent" sentiment="accent" />
+  <sage-chip label="Danger" sentiment="danger" />
+  <sage-chip label="Info" sentiment="info" />
+  <sage-chip label="Neutral" sentiment="neutral" />
+  <sage-chip label="Success" sentiment="success" />
+  <sage-chip label="Warning" sentiment="warning" />
+</Canvas>
+
+### Dots
```

Comment:
Can `Dots` be updated to `Dot`? Throughout the docs, this is the only instance of plural use.